packages/utils/src/lib/log-results.ts

```diff
@@ -29,14 +30,20 @@ export function logMultipleResults<T>(
 
 export function logPromiseResults<
   T extends PromiseFulfilledResult<unknown> | PromiseRejectedResult,
->(results: T[], logMessage: string, callback: (result: T) => void): void {
+>(results: T[], logMessage: string, getMsg: (result: T) => string): void {
   if (results.length > 0) {
-    if (results[0]?.status === 'fulfilled') {
-      console.info(logMessage);
-    } else {
-      console.warn(logMessage);
-    }
+    const log =
+      results[0]?.status === 'fulfilled'
+        ? (m: string) => {
+            ui().logger.success(m);
+          }
+        : (m: string) => {
+            ui().logger.warning(m);
+          };
```

Comment:
```suggestion
    const log =
      results[0]?.status === 'fulfilled'
        ? ui().logger.success
        : ui().logger.warning;
```

Reply:
The suggested change wasn't made, but was resolved without comment :confused: 

![image](https://github.com/code-pushup/cli/assets/34691111/d2d72237-e7db-4027-9967-56023150d058)

Reply:
Sry!! This breaks because of `this`scope.

Reply:
Ah, OK. Thanks for explaining.